tests/backend/user/test_apiv2_human_find.py

```diff
@@ -311,3 +323,14 @@ async def test_bad_args(access_testbed, organization_factory, local_device_facto
     for bad in [{"page": 0}, {"per_page": 0}, {"per_page": 101}]:
         rep = await human_find(sock, **bad)
         assert rep["status"] == "bad_message"
+
+
+@pytest.mark.trio
+async def test_human_handle_to_none(access_testbed, organization_factory, local_device_factory):
+    binder, org, godfrey1, sock = access_testbed
+
+    richard = local_device_factory(base_human_handle=None, org=org)
```

Comment:
you can use the awesome `@customize_fixtures(alice_has_human_handle=False)` decorator on the test to have fixture `alice` and `alice2` without human_handle (works also for bob and adam) !

Reply:
done